include/sdf/Link.hh

```diff
@@ -192,6 +192,16 @@ namespace sdf
     /// \sa const ignition::math::Inertiald &Inertial() const
     public: bool SetInertial(const ignition::math::Inertiald &_inertial);
 
+    /// \brief Resolve the Inertial pose to a specified frame.
```

Comment:
This says resolve the inertial pose, but we're not resolving the pose are we?

Reply:
https://github.com/gazebosim/sdformat/pull/1012/commits/8d7355996359a0271e873360d85df7f6c3172d26